feature/experimental/policy/policy_base/base_vrf_selection_test.go

```diff
@@ -0,0 +1,393 @@
+)
+
+var (
+	ateSrc = attrs.Attributes{
+		Name:    "ATE to Ingress Source",
+		IPv4:    "192.0.2.2",
+		IPv6:    "2001:db8::2",
+		IPv4Len: plen4,
+		IPv6Len: plen6,
+	}
+	dutSrc = attrs.Attributes{
+		Desc:    "Ingress to ATE Source",
+		IPv4:    "192.0.2.1",
+		IPv6:    "2001:db8::1",
+		IPv4Len: plen4,
+		IPv6Len: plen6,
+	}
+	ateDst = attrs.Attributes{
+		Name:    "ATE to Egress VLAN 10",
+		IPv4:    "192.0.2.6",
+		IPv6:    "2001:db8::6",
+		IPv4Len: plen4,
+		IPv6Len: plen6,
+	}
+	dutDst = attrs.Attributes{
+		Desc:    "Egress VLAN 10 to ATE",
+		IPv4:    "192.0.2.5",
+		IPv6:    "2001:db8::5",
+		IPv4Len: plen4,
+		IPv6Len: plen6,
+	}
+	ateDst2 = attrs.Attributes{
+		Name:    "ATE to Egress VLAN 20",
+		IPv4:    "192.0.2.10",
+		IPv6:    "2001:db8::a",
+		IPv4Len: plen4,
+		IPv6Len: plen6,
+	}
+	dutDst2 = attrs.Attributes{
+		Desc:    "Egress VLAN 20 to ATE",
+		IPv4:    "192.0.2.9",
+		IPv6:    "2001:db8::9",
+		IPv4Len: plen4,
+		IPv6Len: plen6,
+	}
+)
+
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice, p1 *ondatra.Port, p2 *ondatra.Port) {
+	d := dut.Config()
+
+	// Configure ingress interface
+	t.Logf("*** Configuring interfaces on DUT ...")
+	i1 := &telemetry.Interface{Name: ygot.String(p1.Name())}
+	d.Interface(p1.Name()).Replace(t, configInterfaceDUT(i1, &dutSrc, &ateSrc, 0, 0))
+
+	// Configure egress interface
+	i2 := &telemetry.Interface{Name: ygot.String(p2.Name())}
+	d.Interface(p2.Name()).Replace(t, configInterfaceDUT(i2, &dutDst, &ateDst, 1, vlan10))
+	d.Interface(p2.Name()).Replace(t, configInterfaceDUT(i2, &dutDst2, &ateDst2, 2, vlan20))
+
+	// Configure network instance
+	t.Logf("*** Configuring network instance on DUT ... ")
+	niConfPath := dut.Config().NetworkInstance("10")
+	niConf := configNetworkInstance("10", &ateDst)
+	niConfPath.Replace(t, niConf)
+	niConfPath = dut.Config().NetworkInstance("20")
+	niConf = configNetworkInstance("20", &ateDst2)
+	niConfPath.Replace(t, niConf)
+
+	// Configure default NI and forwarding policy
+	t.Logf("*** Configuring default instance forwarding policy on DUT ...")
+	dutConfPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance)
+	dutConfPath.Type().Replace(t, telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+	policyDutConf := configForwardingPolicy()
+	dutConfPath.PolicyForwarding().Replace(t, policyDutConf)
+}
+
+func configInterfaceDUT(i *telemetry.Interface, me, peer *attrs.Attributes, subintfindex uint32, vlan uint16) *telemetry.Interface {
+	i.Description = ygot.String(me.Desc)
+	i.Type = telemetry.IETFInterfaces_InterfaceType_ethernetCsmacd
+	if *deviations.InterfaceEnabled {
+		i.Enabled = ygot.Bool(true)
+	}
+
+	// Create subinterface
+	s := i.GetOrCreateSubinterface(subintfindex)
+
+	if vlan != 0 {
+		// Add VLANs
+		singletag := s.GetOrCreateVlan().GetOrCreateMatch().GetOrCreateSingleTagged()
+		singletag.VlanId = ygot.Uint16(vlan)
+	}
+	// Add IPv4 stack
+	s4 := s.GetOrCreateIpv4()
+	if *deviations.InterfaceEnabled {
```

Comment:
Please guard change this to  `if *deviations.InterfaceEnabled && !*deviations.IPv4MissingEnabled`

Reply:
Done, in new commit. Thank You.